CMakeLists.txt

```
@@ -93,6 +93,10 @@ SET(${PROJECT_NAME}_EXCLUDE_DISABLED_SUBPACKAGES_FROM_DISTRIBUTION_DEFAULT FALSE
 
 SET(Trilinos_USE_GNUINSTALLDIRS_DEFAULT ON)
 
+SET(Trilinos_MUST_FIND_ALL_TPL_LIBS_DEFAULT TRUE)
+
+SET(CMAKE_LINK_LIBRARIES_ONLY_TARGETS)
```

```suggestion
SET(CMAKE_LINK_LIBRARIES_ONLY_TARGETS TRUE)
```
Fun fact, the command:

```
SET(CMAKE_LINK_LIBRARIES_ONLY_TARGETS)
```

actually **unsets** the variable `CMAKE_LINK_LIBRARIES_ONLY_TARGETS` if it is already set in the current scope (but does not unset a cache var of the same name).  See:

* https://cmake.org/cmake/help/latest/command/set.html#normal

which says:

> If no value is given, unset the variable. This is equivalent to [unset(<variable>)](https://cmake.org/cmake/help/latest/command/unset.html#command:unset).

Pretty crazy, right?

---

Would it make a difference if I promised that I typed ON there before and that obviously my computer messed it up, not me?  Fixed.